test/tests/kernels/2d_diffusion/tests

```diff
@@ -70,4 +70,14 @@
     issues = '#11760'
     design = 'FEProblem.md'
   [../]
+
+  [./test_hypre_matrix]
```

Comment:
This is not a good place to add this test. Why not add this to a "solvers" or "preconditioners" folder?